api/views/admin/admin.py

```diff
@@ -803,3 +716,10 @@ def hash_pw(password):
     if sys.version_info >= (3, 4):
         password = password.encode(encoding='utf-8', errors='strict')
     return hashlib.sha256(password).hexdigest()
+
+
+def get_class_by_tablename(name):
+    with current_app.app_context():
+        for mapper in db.Model.registry.mappers:
+            if mapper.class_.__tablename__ == name and hasattr(mapper.class_, '__tablename__'):
+                return mapper.class_
```

Comment:
new line.

Note that it is a good style to always put the newline as a last character if it is allowed by the file format